doc/TUTORIAL_SCRIPT.md

```diff
@@ -0,0 +1,51 @@
+# Script
+
+```shell script
+docker run -it elektra/elektra                                      > NEW LINE
+
```

Comment:
start here with an echo that mount will map the content from hello.js to Elektra's configuration in /hello.